src/examples/run-reflect-tests.yml

```diff
@@ -1,9 +1,9 @@
 description: >
-  Executes a Reflect test suite.
+  Run end-to-end tests using Reflect's no-code testing platform.
 usage:
   version: 2.1
   orbs:
-    reflect: reflect/reflect-circleci-orb@0.1.0
+    reflect: reflect/reflect@0.1.0
```

Comment:
The next release of this orb will likely be `0.2.0` or `1.0.0`, the version shown here in the description should match what the next release of the orb will be.

Reply:
Fixed